data/reusables/repositories/request-security-advisory-cve-id.md

```diff
@@ -1,5 +1,3 @@
-If you don't already have a CVE identification number for the security vulnerability in your project, you can request a CVE identification number from {% data variables.product.prodname_dotcom %}. {% data variables.product.prodname_dotcom %} usually reviews the request within 72 hours. Requesting a CVE identification number doesn't make your security advisory public. If your security advisory is eligible for a CVE,  {% data variables.product.prodname_dotcom %} will reserve a CVE identification number for your advisory. We'll then publish the CVE details after you publish the security advisory.
+If you don't have a CVE identification number for the security vulnerability in your project and need one you can request a CVE identification number from {% data variables.product.prodname_dotcom %}. {% data variables.product.prodname_dotcom %} usually reviews the request within 72 hours. If your security advisory is eligible for a CVE, {% data variables.product.prodname_dotcom %} will reserve a CVE identification number for your advisory. We'll then publish the CVE details after you make public you security advisory. It should be noted that anyone with admin permissions to a security advisory can request a CVE identification number for a security advisory.
```

Comment:
A few rewordings, plus I think it's important to keep the mention that requesting a CVE doesn't make things public.

```suggestion
If you want a CVE identification number for the security vulnerability in your project, and don't already have one, you can request a CVE identification number from {% data variables.product.prodname_dotcom %}. {% data variables.product.prodname_dotcom %} usually reviews the request within 72 hours. Requesting a CVE identification number doesn't make your security advisory public. If your security advisory is eligible for a CVE, {% data variables.product.prodname_dotcom %} will reserve a CVE identification number for your advisory. We'll then publish the CVE details after you make your security advisory public. Anyone with admin permissions to a security advisory can request a CVE identification number.
```